dir.props

```diff
@@ -193,10 +193,10 @@
     <!-- same thing here for CLI -->
     <ExtraPackageVersionPropsPackageInfo Include="MicrosoftNETCoreAppRuntimewinx64Version" Version="$(coresetupOutputPackageVersion)" />
     <!-- same thing here for toolset -->
-    <ExtraPackageVersionPropsPackageInfo Include="MicrosoftNETCoreAppRuntimewinx64PackageVersion" Version="$(coresetupOutputPackageVersion)" />
+    <ExtraPackageVersionPropsPackageInfo Include="MicrosoftNETCoreAppRuntimewinx64PackageVersion" Version="%24(MicrosoftNETCoreDotNetAppHostPackageVersion)" />
```

Comment:
I don't understand what this change is doing.

Reply:
Toolset uses `MicrosoftNETCoreAppRuntimewinx64PackageVersion` to decide what version of `Microsoft.NETCore.App` it uses because `MS.NETCore.App` is no longer a real package.  We were previously setting this version to `$(coresetupOutputPackageVersion)`, which is our variable for the version of core-setup we are building and is taken verbatim from Version.Details.xml.  This broke due to stabilization because core-setup always includes the build suffix on its output version, but produces some packages without the suffix when `UseStableVersions` is turned on.  `MicrosoftNETCoreDotNetAppHostPackageVersion` matches the version of `MS.NETCore.App`, including whether it includes the suffix or not, so this just copies that version instead of supplying our own.

Reply:
Got it.  Thanks for explaining!